src/components/track-tile/TrackTileCoSign.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+
+import { Remix } from 'audius-client/src/common/models/Track'
+import { StyleSheet, View } from 'react-native'
+
+import Text from 'app/components/text'
+import UserBadges from 'app/components/user-badges/UserBadges'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { flexRowCentered } from 'app/styles'
+import { ThemeColors } from 'app/utils/theme'
+
+import { createStyles as createTrackTileStyles } from './styles'
+
+const messages = {
+  coSign: 'Co-Sign',
+  reposted: 'Reposted',
+  favorited: 'Favorited',
+  repostedAndFavorited: 'Reposted & Favorited'
+}
+
+const formatCoSign = ({
+  hasReposted,
+  hasFavorited
+}: {
+  hasReposted: boolean
+  hasFavorited: boolean
+}) => {
+  if (hasReposted && hasFavorited) {
+    return messages.repostedAndFavorited
+  } else if (hasFavorited) {
```

Comment:
something that a linter could automate, but technically unecessary else if if we are returning in each block